css/main.css

```diff
@@ -325,3 +325,69 @@ li {
 .skills {
   list-style: inside url("../media/svg/circle.svg");
 }
+
+@media screen and (max-width: 700px) {
+  .formtext {
+    justify-content: center;
+    display: flex;
+    text-align: center;
+    color: #172b4d;
+    font-weight: 700;
+    font-size: 32px;
+    line-height: 30px;
+    padding: 1.5rem;
+  }
+
+  /* .firsttext {
+  } */
+  .forml .namepart {
```

Comment:
Don't forget to align the input boxes please :triangular_ruler: 

[Here](https://stackoverflow.com/questions/54304337/html-css-align-boxes-of-different-sizes) are various ways to do so! :partying_face: